Bio/MarkovModel.py

```diff
@@ -43,6 +44,7 @@ def logaddexp(logx, logy):
 
 
 def itemindex(values):
+    """Return a dictionary that hold values and their position inside the sequence as key."""
```

Comment:
hold --> holds.

Maybe `` """Return a dictionary of values with their sequence offset as keys."""`` is clearer?